android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/TopAppBars.kt

```diff
@@ -161,8 +161,10 @@ fun TaskDetailTopAppBar(onBack: () -> Unit, onDelete: () -> Unit) {
             }
         },
         actions = {
-            IconButton(onClick = onDelete) {
-                Icon(Icons.Filled.Delete, stringResource(id = R.string.menu_delete_task))
+            if (StatsIgUtil.isDeleteTodoFeatureEnabled()) {
```

Comment:
just add `Statsig.checkGate` here

Reply:
Okay. I have removed the StatsigUtil and replace with Statsig.checkGate